Components/WebService/Rest/RestConfigurableProtocol.php

```diff
@@ -58,6 +60,7 @@ public function configureOptionsResolver(OptionsResolver $resolver)
             self::OPTION_HEADERS => [],
             self::OPTION_AUTH => '',
             self::OPTION_ENCODING => self::ENCODING_JSON,
+            self::OPTION_REQUEST_FALLBACK_ON_ERROR => true,
```

Comment:
On the part below of this method we should define that the allow type for `OPTION_REQUEST_FALLBACK_ON_ERROR` should be `'boolean'`

Reply:
Good idea, added.